### DISCUSSION TITLE
Can I use the search feature from flash.nvim (`s`) when NeoTree is focused?

### DISCUSSION THREAD

**Discussion by GerritWanderer** [LazyVim/LazyVim#4677] [Q&A] [answered]
Hi LazyVim Community,

it seems like I can only activate the search feature from flash.vim (`s`) from inside of a file buffer.
And if I want to use the same search feature when the Neotree window is focused, it opens the current selected folder instead of the search feature. Is this a known limitation and is it possible to enable the searchfor Neotree window as well?

The only workaround I found so far is to to activate the search from my file buffer while Neotree is visible too. This will then expand the search to all open windows, however this is not always the fastest way and comfortable way.
A search for similar discussion have not brought up any result unfortunately.

Best
Gerrit

**Comment by dpetka2001** [comment, marked as answer]
That's because they use both conflicting mappings for different actions. You could try the following
```lua
  {
    "nvim-neo-tree/neo-tree.nvim",
    opts = {
      window = {
        mappings = {
          ["s"] = false,
          ["another_key"] = "open_vsplit",
        },
      },
    },
  },
```
Disable `s` key in neo-tree, so that the default flash.nvim key will be used and use another key of your liking for opening in vertical split.

**Comment by GerritWanderer** [comment]
Thank you so much @dpetka2001 , works like a charm with your suggestion do disable the key mapping for `s` 🙌
